packages/ui/src/utils/typography.ts

```diff
@@ -61,4 +61,4 @@ export const button = cn('font-default text-textBase font-medium leading-textBas
 
 export const buttonMedium = cn('font-default text-textBase font-medium leading-textBase');
 
-export const buttonSmall = cn('font-default text-textBase font-medium leading-textBase');
+export const buttonSmall = cn('font-default text-textXs font-medium leading-textBase');
```

Comment:
issue: it's `textBase` in Figma

<img width="1224" alt="image" src="https://github.com/user-attachments/assets/fdc1db21-4c1c-4aec-8dfb-8b11838ae742" />

Reply:
I think that's out of date, see here:
<img width="468" alt="image" src="https://github.com/user-attachments/assets/38f5d47c-9134-4f96-9fb6-3af07e9a7ad8" />

Reply:
There's also the mobile size, which is not yet supported in the frontend

<img width="876" alt="image" src="https://github.com/user-attachments/assets/c6693767-88fc-4c9e-a695-c33a7541394f" />